pkg/sysinfo/numMem_linux.go

```diff
@@ -0,0 +1,7 @@
+// +build linux
```

Comment:
What? I know for a fact Linux supports NUMA...

Reply:
The `SYS_NUMA_GETAFFINITY` const doesn't seem to exist when built for linux. Is it under another name?

Reply:
I do see a `get_mempolicy` for Linux, https://man7.org/linux/man-pages/man2/get_mempolicy.2.html - `MPOL_F_MEMS_ALLOWED` looks like what we want?

Reply:
ok that is what I tried initially but only returned errors, will try to rework

Reply:
@mheon what is the value of/what do we have `MPOL_F_MEMS_ALLOWED` stored as?